datreant/collections.py

```diff
@@ -1,23 +1,23 @@
 """
-The Bundle object is the primary manipulator for Treants in aggregate.
-They are returned as queries to Groups, Coordinators, and other Bundles. They
-offer convenience methods for dealing with many Treants at once.
+The Bundle object is the primary manipulator for Treants in aggregate. They are
+returned as queries to Groups and other Bundles. They offer convenience methods
+for dealing with many Treants at once.
 
 """
 import os
 
-import numpy as np
 import multiprocessing as mp
 import glob
 import fnmatch
 
+from six import string_types
```

Comment:
also do a `from six.moves import zip` because you use zip later in this file. It will make sure you are using izip in python2 and 3.